backend/app/model/mixins/representative_file_version.rb

```diff
@@ -1,6 +1,33 @@
-# mixin for archival records that
-# need a computed "representative file version" for
-# banner images and thumbnails in public interface etc.
+# This module calculates and applies if possible a read-only #representative_file_version
+# property to the following Record Types: DigitalObject, DigitalObjectComponent, Accession,
+# Resource and ArchivalObject. It is intended to encapsulate all the logic for how
+# representations are selected and displayed per this document:
+#
+# https://docs.google.com/document/d/1-fi-1jpE7GqWc0zWxWsRO9zgRmR1vFFygp6jGnKh23k/edit?usp=sharing
```

Comment:
Minor: put the spec on the wiki or in tech docs and link to that?

Reply:
I changed the link to the JIRA issue.